.travis.yml

```diff
@@ -10,3 +10,16 @@ script:
   - cargo test --verbose
   - cargo build --verbose --examples
 
+jobs:
+  include:
+    - stage: integration
+      env:
+        - VERSION=0.18.0
+      install:
+        - wget https://bitcoincore.org/bin/bitcoin-core-$VERSION/bitcoin-$VERSION-x86_64-linux-gnu.tar.gz
+        - tar -xzf bitcoin-$VERSION-x86_64-linux-gnu.tar.gz
+        - cd bitcoin-$VERSION/bin
+        - ./bitcoind -regtest &
+      rust: stable
+      script:
+        - cargo run --example regtest
```

Comment:
I wouldn't run example, I'd have a `tests/integration.rs` that can be just one really long test accessing tons of methods in sequence. It's probably easier that way than to use individual tests because ordering can't be enforced.

Reply:
I thought about that, but how easy is it to deactivate this one test during normal test runs without bitcoind?

Reply:
Ah you want only one Travis build to do the integration test? Hmm, no idea.

Reply:
Yeah, you don't want normal unit tests to fail if no regtest daemon is running. So I'd also separate both test suites on travis. That also makes it easier to run the integration tests against multiple bitcoind versions without the build matrix growing too much.
![test_pipeline](https://user-images.githubusercontent.com/1876998/61410394-788c9600-a8e4-11e9-956a-4ad582a71a0b.png)

Reply:
If you the `docker` service in travis, you could use our bitcoind testcontainer to spin it up. Example here:

https://github.com/testcontainers/testcontainers-rs/blob/6f761608eb48454ba28591db3ca837b1369f9055/tests/images.rs#L20-L40